src/cluster_legacy.h

```diff
@@ -138,6 +138,8 @@ typedef enum {
     CLUSTERMSG_EXT_TYPE_HUMAN_NODENAME,
     CLUSTERMSG_EXT_TYPE_FORGOTTEN_NODE,
     CLUSTERMSG_EXT_TYPE_SHARDID,
+    CLUSTERMSG_EXT_TYPE_CLIENT_IPv4,
```

Comment:
Should we capitalize `V`?
```suggestion
    CLUSTERMSG_EXT_TYPE_CLIENT_IPV4,
```

Reply:
If you insist... I thought it looks pretty like this. :)

Reply:
Changed.